src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -0,0 +1,935 @@
+                statistics.put("Active Conference#Conference system", getJsonProperty(terminal.get(0), "systemID").asText());
+            }
+        }
+        if (connections != null) {
+            statistics.put("Active Conference#Conference type", getJsonProperty(connections, "callType").asText());
+            statistics.put("Active Conference#Conference info", getJsonProperty(connections, "callInfo").asText());
+        }
+        statistics.put("Active Conference#Conference start time", new Date(getJsonProperty(activeConference, "startTime").asLong()).toString());
+        return getJsonProperty(activeConference, "id").asInt();
+    }
+
+    /**
+     * Retrieve list of all current conference calls
+     *
+     * @return ArrayNode containing the list of conference calls
+     * @throws Exception during http communication
+     */
+    private ArrayNode getAllConferenceCalls() throws Exception {
+        return doGet(buildHttpUrl(CONFERENCE), ArrayNode.class);
+    }
+
+    /**
+     * Retrieve device's active call media stats. Device does not immediately release the device when the hangup
+     * is requested so it may end up heve while is not in the call anymore, so 404 error would indicate that there's
+     * no such conference anymore, but there has been one since the conferenceId was retrieved.
+     *
+     * @return ArrayNode containing RX/TX stats
+     * @throws Exception during http communication
+     */
+    public ArrayNode retrieveConferenceCallMediaStats(int conferenceId) throws Exception {
+        try {
+            return doGet(buildHttpUrl(String.format(MEDIASTATS, conferenceId)), ArrayNode.class);
+        } catch (CommandFailureException cfe) {
+            if (cfe.getStatusCode() == 404) {
+                if(logger.isDebugEnabled()) {
+                    logger.debug("Conference " + conferenceId + " is not available anymore. Skipping media stats retrieval.");
+                }
+                return JsonNodeFactory.instance.arrayNode();
+            } else {
+                logger.error("Unable to find conference by id " + conferenceId);
+                throw cfe;
+            }
+        }
+    }
+
+    /**
+     * Request device reboot
+     *
+     * @throws Exception during http communication
+     */
+    private boolean reboot() throws Exception {
+        ObjectNode rebootRequest = JsonNodeFactory.instance.objectNode();
+        rebootRequest.put("action", "reboot");
+        boolean success = doPost(REBOOT, rebootRequest).getStatusCode().is2xxSuccessful();
+        if (success) {
+            sessionId = null;
+            internalDestroy();
+        }
+        return success;
+    }
+
+    /**
+     * Retrieve device system information, including hardware info and lan status
+     *
+     * @param statistics map to put values to
+     * @throws Exception during http communication
+     */
+    public void getSystemInfo(Map<String, String> statistics) throws Exception {

```

Comment:
Is there a reason this method is public? also, name should start with "retrieve" instead of "get"